_content/av/can-everyone-do-the-full-lotus-pose_akram-yoga.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Can everyone do the full lotus pose?"
+authors:
+  - "Akram Yoga & Teacher Training
+external_url: "https://www.youtube.com/watch?v=FfAJKjn2CjY"
+drive_links:
+  - "https://docs.google.com/document/d/1zok4vrUg_cC5gX10PTrVJDaUspwOSdzYCaK7lIN2NjE/edit?usp=drive_link"
```

Comment:
These docs that are just links don't need to be included. Just have an external url and that's enough